atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/SweepStatsKeyValueService.java

```diff
@@ -218,7 +179,9 @@ public void run() {
                         }
                     }
                 } catch (Throwable t) {
-                    log.error("Error occurred while flushing sweep stats: {}", t, t);
+                    if (!Thread.interrupted()) {
```

Comment:
Should this be `if (!Thread.currentThread().isInterrupted()) {` to avoid clearing interrupt flag?